cypress/plugins/index.js

```diff
@@ -1,22 +1,2 @@
-const fs = require("fs-extra");
```

Comment:
Can this whole file `cypress/plugins/index.js` be removed now? It doesn't seem to be imported by cypress.config.ts above.

Reply:
@scytacki , I've removed `plugins/index.js` now. Cypress has deprecated it in their newer versions.